### PR TITLE
[FEATURE] retrieve the log level

### PR DESCRIPTION
**Pull request by iariasleon**
```

                    SUMMARY:
-------------------------------------------------
  - components\ngsiv2\admin\retrieve_log_level\retrieve_log_level.feature >> passed: 1, failed: 0, skipped: 0 and total: 1 with duration: 0.098 seconds.
-------------------------------------------------
  Date: Tuesday, August 02, 2016 12:06:50
-------------------------------------------------
1 feature passed, 0 failed, 0 skipped
1 scenario passed, 0 failed, 0 skipped
3 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m0.098s
```
@fgalan @kzangeli @crbrox 
issue associated #2352

### CODE REVIEW

File: test/acceptance/behave/components/common_steps/general_steps.py [hunk 1]
```diff
@@ -342,3 +354,15 @@ def verify_headers_in_response(context):
     ngsi = NGSI()
     ngsi.verify_headers_response(context)
     __logger__.info("...Verified headers in response")
+
+
+@step(u'verify if the log level "([^"]*)" is the expected')
+def verify_if_the_log_level_is_the_expected(context, level):
+    """
+    verify if the log level is the expected
+    :param context: It’s a clever place where you and behave can store information to share around. It runs at three levels, automatically managed by behave.
```
Comment (fgalan) [inline test/acceptance/behave/components/common_steps/general_steps.py]: level param is missing in docstring

File: test/acceptance/behave/components/ngsiv2/admin/retrieve_log_level/retrieve_log_level.feature [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Feature: retrieve the log level in Context Broker
+  As a context broker user
+  I would like to retrieve the log level in Context Broker
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/admin/retrieve_log_level/retrieve_log_level.feature]: This step starts CB at DEBUG level by default?

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/admin/retrieve_log_level/retrieve_log_level.feature]: Yes, CB starts by default with this config `"-multiservice -t 0-255 -logLevel DEBUG"`

File: test/acceptance/behave/components/common_steps/general_steps.py (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/general_steps.py]: Fixed in 720bb6d

File: test/acceptance/behave/components/ngsiv2/admin/retrieve_log_level/retrieve_log_level.feature (hunk 2 above)
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/admin/retrieve_log_level/retrieve_log_level.feature]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**